packages/bogue/bogue.20190717/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "-p" name "@doc"] {with-doc}
+]
+maintainer: ["Vu Ngoc San <san.vu-ngoc@laposte.net>"]
+authors: ["Vu Ngoc San <san.vu-ngoc@laposte.net>"]
+bug-reports: "https://github.com/sanette/bogue/issues"
+homepage: "https://github.com/sanette/bogue"
+doc: "http://sanette.github.io/bogue/Bogue.html"
+license: "GPL2"
+dev-repo: "git+https://github.com/sanette/bogue.git"
+synopsis: "GUI library for ocaml, with animations, based on SDL2"
+description: """
+bogue is a GUI library for ocaml, with animations, based on SDL2.
+
+This library can be used for games or for adding GUI elements to any
+ocaml program.
+
+It uses the SDL2 renderer library, which makes it quite fast.
+
+It is themable, and does not try to look like your desktop. Instead,
+it will look the same on every platform.
+
+Graphics output is scalable, and hence easily adapts to Hi-DPI
+displays.
+
+Programming with bogue is easy if you're used to GUIs with widgets,
+layouts, callbacks, and of course it has a functional flavor. It uses
+Threads when non-blocking reactions are needed."""
+depends: [
+  "tsdl-image" {>= "0.2"}
+  "tsdl-ttf" {>= "0.2"}
+  "ocaml" {>= "4.03.0"}
+  "dune" {build & > "1.10"}
```

Comment:
```suggestion
  "dune" {>= "1.10"}
```
Dune should not be tagged as a build dependency anymore. Also you probably want `>= 1.10` not `> 1.10`